scripts/_commands.sh

```diff
@@ -11,16 +11,46 @@ function menu() {
   /home/joinmarket/menu.sh
 }
 
+# command: newnym
+function newnym(){
+  if [ "$(cat /home/joinmarket/joinin.conf 2>/dev/null | grep -c "runBehindTor=on")" -eq 1 ]; then
+    echo "Changing Tor circuits..."
+    echo "Savind old ID..."
+    oldID=$(curl --connect-timeout 15 --socks5-hostname 127.0.0.1:9050 ifconfig.me 2>/dev/null)
+    echo "Requesting new identity ..."
+    sudo python3 /home/joininbox/scripts/standalone/tor.newnym.py
```

Comment:
Should be running with a non-root user having access to the tor-auth-cookie.
`sudo -u debian-tor` or `bitcoin` (part of the `debian-tor` group) would be a good choice.